docs/OnTrack/Deployment/Google Cloud/Research & Findings/google-cloud-research.md

```diff
@@ -121,7 +121,7 @@ the setup and changes required.
 ## Outcomes
 
 Following the team's research of Google Cloud and its services, deployments, and Docker containers,
```

Comment:
Are the links to the research?

Reply:
The links and references on this page refer back to vendor documentation that we read (such as Docker, Google Cloud, etc.) as well as the Doubtfire deployment documentation in our Thoth Tech repository.